identity-service/src/utils/relayHelpers.js

```diff
@@ -39,8 +89,76 @@ async function doesUserHaveSocialProof (userInstance) {
   return !!twitterUser || !!instagramUser
 }
 
+/**
+ * Derives the authority accounts for the programs using values from the config
+ * @returns {Promise<{solanaClaimableTokenAuthority: string, solanaRewardsManagerAuthority: string}>}
+ */
+const deriveAuthorities = async () => {
```

Comment:
Made a notion doc here with my understanding thus far of how these things get named/piped around. (WIP): https://www.notion.so/audiusproject/Instructions-things-fb41de0620a747a389373ab19cc2f81e